intelmq-mailgen

```diff
@@ -166,22 +268,67 @@ def format_malware_as_csv(events):
     return ("events.csv", contents.getvalue(), "text", "csv")
 
 
-known_formats = {
-    ("csv", "malware"): (malware_columns, format_malware_as_csv),
+
+
+def mail_format_malware_as_csv(cur, notification, config):
+    """Creates one email with csv attachment for malware.
+
+    :returns: list of email objects
+    :rtype: list
+    """
+    attachments = []
+
+    event_attachment = format_malware_as_csv(
+                load_events(cur, malware_columns, notification["event_ids"])
+                )
+
+    if event_attachment is not None:
+        attachments.append(event_attachment)
+
+    subject, body = read_template(config["template_dir"],
+                                  notification["template"])
+
+    mail = create_mail(sender=config["sender"],
+                                  recipient=notification["email"],
+                                  subject=subject, body=body,
+                                  attachments=attachments)
+    return [mail]
```

Comment:
No, this is indentation I like, I'll ask you offline for the style what we want to use.